frontend/src/lib/components/HelpButton/HelpButton.tsx

```diff
@@ -145,19 +147,17 @@ export function HelpButton({
                                 Read the docs
                             </LemonButton>
                         )}
-                        {!isPromptVisible && (
-                            <LemonButton
-                                icon={<IconTrendingUp />}
-                                status="stealth"
-                                fullWidth
-                                onClick={() => {
-                                    promptAction('activation-checklist')
-                                    hideHelp()
-                                }}
-                            >
-                                How to be successful with PostHog
-                            </LemonButton>
-                        )}
+                        <LemonButton
+                            icon={<IconTrendingUp />}
+                            status="stealth"
+                            fullWidth
+                            onClick={() => {
+                                toggleActivationSideBar()
+                                hideHelp()
+                            }}
+                        >
+                            Quick Start using PostHog
```

Comment:
```suggestion
                            Quick Start
```
'Using' just sounds weird here to me. And it's a little redundant.